app/src/main/java/org/mifos/mobile/ui/update_password/UpdatePasswordScreen.kt

```diff
@@ -0,0 +1,230 @@
+            supportingText = confirmPasswordErrorContent,
+            icon = R.drawable.ic_lock_black_24dp,
+            trailingIcon = {
+                var image = if (viewConfirmPassword) Icons.Filled.Visibility
+                else Icons.Filled.VisibilityOff
+
+                if (!confirmPasswordError) {
+                    IconButton(onClick = { viewConfirmPassword = !viewConfirmPassword }) {
+                        Icon(imageVector = image, contentDescription = "password visibility Button")
+                    }
+                } else {
+                    Icon(imageVector = Icons.Filled.Error, contentDescription = null)
+                }
+            },
+            error = confirmPasswordError,
+            visualTransformation = if (viewConfirmPassword) VisualTransformation.None else PasswordVisualTransformation(),
+            keyboardType = KeyboardType.Password
+
+        )
+
+        Button(
+            onClick = {
+                keyboardController?.hide()
+
+                //sets error messages if there is any
+                newPasswordErrorContent = viewModel.getNewPasswordError(newPassword.text)
+                confirmPasswordErrorContent = viewModel.getConfirmPasswordError(confirmPassword.text)
+
+                when {
+                    newPasswordErrorContent.isEmpty() && confirmPasswordErrorContent.isEmpty() -> {
+                        if (viewModel.validatePasswordMatch(newPassword.text, confirmPassword.text)
+                        ) {
+                            if (viewModel.isNetworkConnected()) {
+                                viewModel.updateAccountPassword(
+                                    newPassword.text,
+                                    confirmPassword.text
+                                )
+                            } else Toast.makeText(context, R.string.no_internet_connection, Toast.LENGTH_SHORT).show()
+                        } else Toast.makeText(context, R.string.error_password_not_match, Toast.LENGTH_SHORT).show()
+                    }
+
+                    newPasswordErrorContent.isEmpty() && confirmPasswordErrorContent.isNotEmpty() -> {
+                        confirmPasswordError = true
+                    }
+
+                    newPasswordErrorContent.isNotEmpty() && confirmPasswordErrorContent.isEmpty() -> {
+                        newPasswordError = true
+                    }
+
+                    else -> {
+                        confirmPasswordError = true
+                        newPasswordError = true
+                    }
+                }
+            },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 16.dp, end = 16.dp, top = 4.dp),
+            contentPadding = PaddingValues(12.dp),
+            colors = ButtonDefaults.buttonColors(
+                containerColor = if (isSystemInDarkTheme()) Color(
+                    0xFF9bb1e3
+                ) else Color(0xFF325ca8)
```

Comment:
check if these colors are defined in Color.kt , if not then add them over there and then use here